content/en/continuous_integration/tests/setup/python.md

```diff
@@ -25,13 +25,19 @@ further_reading:
 
 ## Compatibility
 
-Supported Python interpreters:
-* Python >= 3.7
+Supported languages:
+
+| Language | Version |
+|---|---|
+| Python 3 | >= 3.7 |
```

Comment:
This is actually wrong. ITR is only supported in `3.6+` but test visibility's supported down to `2.7`.

Can't remember when this particular section was changed but it shouldn't have.

Reply:
Ohh this happened in https://github.com/DataDog/documentation/pull/19276, I'll open a PR to change it back.

Reply:
```suggestion
| Python 2 | >= 2.7 |
| Python 3 | >= 3.6 |
```